src/Humanizer.Tests.Shared/DateToOrdinalWordsTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using Xunit;
+
+namespace Humanizer.Tests
+{
+    public class DateToOrdinalWordsTests
+    {
+        [Fact]
+        public void CanToOrdinalWords()
+        {
+            var date = new DateTime(2015, 1, 1);
+
+            Assert.Equal("1st January 2015", date.ToOrdinalWords());
+        }
+
+        [UseCulture("pt-BR")]
+        [Fact]
+        public void CanToOrdinalWordsIncludingGender()
+        {
+            var date = new DateTime(2015, 1, 1);
+
+            Assert.Equal("1º janeiro 2015", date.ToOrdinalWords(GrammaticalGender.Masculine));
```

Comment:
It's about the [question on the main thread](https://github.com/Humanizr/Humanizer/pull/500#issuecomment-165405118). For example, in russian the gender of the number _in a date_ is always neutral (средний род), and for some languages it can be different. I don't think that we need this method overload at all. 

But, for russian we need an overload with grammatical case, to say "1-го апреля 2016" (_at_ 1st April 2015), for example.